pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "truss"
-version = "0.9.2rc0"
+version = "0.10.0rc0"
```

Comment:
I know it's not really part of this this PR, but just for curiosity, I've been wondering if we need to support py3.8 or can bump it to at least 3.9?